Content.Server/Chemistry/Components/VaporComponent.cs

```diff
@@ -7,11 +8,19 @@ public sealed partial class VaporComponent : Component
     {
         public const string SolutionName = "vapor";
 
-        [DataField("transferAmount")]
-        public FixedPoint2 TransferAmount = FixedPoint2.New(0.5);
+        /// <summary>
+        /// Stores data on the previously reacted tile. We only want to do reaction checks once per tile.
+        /// </summary>
+        public TileRef PreviousTileRef;
```

Comment:
```suggestion
        public TileRef? PreviousTileRef;
```
should be nullable since it does not have a default value

Reply:
pushed